packages/playground/src/utils/node.ts

```diff
@@ -100,8 +100,10 @@ export async function getNodeAvailability(nodeId: number) {
   if (elapsedSinceLastUptimeEvent >= UPTIME_EVENTS_INTERVAL) {
     downtime += elapsedSinceLastUptimeEvent;
   }
+
+  const node: GridNode = await gridProxyClient.nodes.byId(nodeId);
   console.log(
-    `getNodeAvailability: Node ${nodeId} was down for ${downtime} seconds in the last ${secondsSinceCurrentPeriodStart} seconds.`,
+    `getNodeAvailability: Node ${nodeId} was ${node.status} for ${downtime} seconds in the last ${secondsSinceCurrentPeriodStart} seconds.`,
```

Comment:
The status might be up but we still show the downtime.

Reply:
This function returns the downtime in all of the cases, I also changed only the static node status to be dynamic. what do you suggest?

Reply:
But does this fix anything?

Reply:
Will display the actual status of the node instead of hard-cooding the 'down' status, please check the mentioned issue.

Reply:
Then, I suggest renaming the downtime variable as it's misleading.

Reply:
please check again

Reply:
I opened another issue to update the whole story https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2900